shm/coordinates.cpp

```diff
@@ -0,0 +1,8 @@
+#include "coordinates.hpp"
+
+Coordinates::Coordinates(const int positionX, const int positionY)
+    : positionX_(positionX), positionY_(positionY){};
+
+bool Coordinates::operator==(Coordinates& position) {
```

Comment:
Use const with reference or remove reference. coordinates in not a complex type it is safe to create a copy.

Reply:
Done, I used const with reference